server/services/ApplicationsDeveloper.js

```diff
@@ -130,6 +130,24 @@ class ApplicationsDeveloper {
     return this.request(options);
   }
 
+  updateKey (data, appId, keyId, env) {
+    let restrictedIps = [];
+    let javaScriptDomains = [];
+    restrictedIps = data.restrictedIps.split(",");
+    javaScriptDomains = data.javaScriptDomains.split(",");
```

Comment:
I don't know enough about javascript but do these have to be done on seperate lines?

Reply:
I don't know enough about javascript either. I wouldn't have thought so, but it didn't seem to like it when I initially tried to do it on the same line, they kept coming through as undefined